join.spec.md

```diff
@@ -131,48 +131,42 @@ Documents MUST define a {join__Graph} enum. Each enum value defines a subgraph.
 
 The {join__Graph} enum is used as input to {@join} directives which link fields and types to subgraphs.
 
-# Scalars
-
-Documents MUST include definitions of any scalars they use. Producers SHOULD include the `@specifiedBy` directive, but it is not mandatory.
-
-##! join__Never
+# Directives
 
-:::[definition](photos.graphql#join__Never)
+##! @join__graph
 
-The {join__Never} scalar is used to indicate the type of a field which **cannot be resolved**. This is typically due to a composition error. Producers SHOULD provide more specific error information with an {@error} directive on the relevant field.
+Declare subgraph metadata on {join__Graph} enum values. 
 
-:::[example](photos.graphql#User.favorite) -- Using {join__Never} and {@error} to mark a field error
+```graphql definition
+directive @join__graph(name: String!, url: String!) on ENUM_VALUE
+```
 
-# Directives
+:::[example](photos.graphql#join__Graph) -- Using {@join__graph} to declare subgraph metadata on the {join__Graph} enum values.
 
 ##! @join__type
 
 Join a type to a subgraph, optionally providing an entity key.
```

Comment:
What does "join" mean? Does this mean roughly states that entities of this type can be fetched from the subgraph using the given key? What does it mean to have key be null?

Reply:
Fixed on my branch (though with a TODO about null key)